Cargo.toml

```diff
@@ -12,21 +12,23 @@ tokio = { version = "1.40.0", features = ["full"] }
 anyhow = "1.0.89"
 clap = { version = "4.5.16", features = ["derive"] }
 rand = "0.8.5"
-bincode = "1.3.3"
+bincode = { version = "2.0.0-rc.2", tag = "v2.0.0-rc.2", git = "https://github.com/bincode-org/bincode.git" }
```

Comment:
there is an existing rc-3

Reply:
As we are heavily based on [platinum prover](https://github.com/lambdaclass/lambdaworks/tree/main/provers/cairo), and it's quite unstable, fixing versions matching theirs is better for the moment I think